src/views/presentations.hbs

```diff
@@ -1,15 +1,3 @@
-<html lang="en">
```

Comment:
Strip existing hbs files down to just html fragments, since we now have the concept of a layout.